README.md

```diff
@@ -1,18 +1,49 @@
+
+Note that the Health (FHIR / Argonaut) APIs for access to veteran health records are not yet in production; dev access in sandbox using mock patient data is available, and we expect these APIs to be in production by fall of 2019.
 
 ## Available Services
 
-### Document Upload Service
-- The [VBA document upload service](services/vba_documents) provides a 
-mechanism to submit VBA forms and documents to VA.
+### Address Validation*
+- The [Address Validation](https://developer.va.gov/explore/verification/docs/address_validation) service (for VA internal use only) provides methods to both standardize and validate addresses.
+
+### Appeals Status*
+- Use the [Appeals Status](https://developer.va.gov/explore/benefits/docs/appeals) API to request the status of a Veteran's benefits claim appeal. Currently for VA internal use only, but third-party access is coming soon.
+
+### Authorization (OpenID Connect)
+- For [Health](https://developer.va.gov/explore/health/docs/authorization) and [Veteran Verification](https://developer.va.gov/explore/verification/docs/authorization).  The VA API Platform uses the OpenID Connect standard to allow Veterans to authorize third-party applications to access data on their behalf. 
+
+### Benefits Claims
+- Use the [Claims](https://developer.va.gov/explore/benefits/docs/claims) (Auto-Establishment) API to submit a Veteran's benefits claim via EVSS (Electronic Veterans Self-Service), and to request the status of a Veteran's benefits claim.
+
+### Benefits Intake (fka Document Upload Service)
+- The [Benefits Intake](https://developer.va.gov/explore/benefits/docs/benefits) API allows authorized third-party systems used by Veteran Service Organizations and agencies to upload claim documents (scanned to PDF) directly to the Veterans Benefits Administration's (VBA) claims intake process.
+
+### Disability Rating
+- The [Disability Rating](https://developer.va.gov/explore/verification/docs/disability_rating) API allows third-parties to access the disability rating of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+
+### Facilities
+- Use the [Facilities](https://developer.va.gov/explore/facilities/docs/facilities) API to find contact info, location, hours of operation, available services and other relevant information about a specific VA facility. 
+
+### Health (currently dev only)
+- Use the [Health APIs](https://developer.va.gov/explore/health/docs/argonaut) to allow Veterans to view their medical records, schedule an appointment, find a specialty facility, and share their information with caregivers and providers.
+
+### Service History
+- The [Service History](https://developer.va.gov/explore/verification/docs/service_history) API allows third-parties to access the service history of a Veteran after receiving authorization to do so using an Open ID Connect flow.
+
+### Veteran Confirmation
```

Comment:
Should we also mention the term "Veteran Verification" here to make sure it is consistent with the API listing?

Reply:
@stephaniemace so I've listed all the APIs here alphabetically without regard to their "category" - Verification is the category containing 3 separate APIs: Confirmation, Service History, and Disability Rating.  Likewise, the Benefits category includes Benefits Intake (fka Document Upload), Benefits Claims (Auto-Establishment) and Appeals Status (internal only).  

I think it's clear from the links to the documentation and probably not necessary to mention since Verification is not an actual API but rather a group of them.  This is not a hill I want to die on, however.  I can rearrange into groups if it should match the `/explore` nav more.  Let me know what you think!

Reply:
This isn't entirely accurate. "Veteran Verification" is the API, and it currently has 4 available endpoints. For better matching with the portal, I would list these as ### headings (sorted alphabetically) and then the available endpoints for each as #### headings underneath (also sorted alphabetically).

I would also remove Authorization entirely as it isn't really a service, but instead how the APIs are accessed.

Reply:
got it.  thanks @jeff2d2 !

Reply:
@jeff2d2 Verification only has 3 endpoints, correct?  (Address validation is listed with them on the dev portal but not actually a Verification endpoint.)

Reply:
Technically, that is correct. Address Validation is it's own API. I'm not sure of the history on how it ended up under Veteran Verification on the portal.